setup.py

```diff
@@ -402,12 +402,12 @@ def configure(platform, build_type):
         "htm.examples": ["*.csv"],
     },
     #install extras by `pip install htm.core[examples]`
-    extras_require={'scikit-image>0.15.0':'examples',
-                    'sklearn':'examples',
-                    'matplotlib':'examples',
-                    'PIL':'examples',
-                    'requests':'examples',
-                    'scipy':'examples'
+    extras_require={'examples':['scikit-image>0.15.0',
+                    'sklearn',
+                    'matplotlib',
+                    'pillow',
```

Comment:
thanks for correcting that. The old format used to work, so likely `pip` changed somewhere in the middle. 
Can you please check the advanced tests are run in the CI (I think so) - but that would require this method to be working even before..?

Reply:
These are not needed for tests, only for examples, since these are plotting libs.
So i think it haven't have to work before. BTW i am not experienced with the CI...

Reply:
sklearn is not for plots but it is used also in examples only 
![image](https://user-images.githubusercontent.com/23330229/87399130-33627900-c5b7-11ea-8d56-c934dd95171e.png)